### ISSUE TITLE
Flow Diagram

### ISSUE THREAD

**Issue by Di2g10** [ssddanbrown/BookStack#66]
Not Sure if this takes product away from targeted use case but the ability to create flow diagrams to allow visualisation of a concept.
Specifically this would be A collection of shapes with text inside and Connectors that can have text attached. Along with some positioning aids to keep things strait and lined up.

My thought for implimenting this would be to have option at top to create flow diagram and this opens up and area or seperate window which allows editing but once editing is complete treat like images inline with text. Perhap with a double click to become editable.

**Comment by ssddanbrown** [comment]
This is a great idea but I feel the amount of work required to add in and maintain a decent diagram editor will be way too much and would distract from the real purpose of the project. There are many diagram editors out there which can export to an image/svg that you're then able to embed or link into BookStack page. 

I feel this feature would just be too big and just a bit too far out of scope to be worth implementing. I will close this issue for now but am open to discussion below.

**Comment by ssddanbrown** [comment]
Alternate solution of draw.io integration will be in the next release, As planned here: #632 #619 

Will effectively achieve this request.
